frontend/packages/dev-console/integration-tests/features/monitoring/monitoringa-Page-Smoke-Tests-ODC-4116.feature

```diff
@@ -0,0 +1,101 @@
+Feature: Monitoring Page
+    As a user, I should be able to perform actions related to monitoring a project and deployments inside it
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke, @regression
+Scenario: Monitoring Page: MH-03-TC02
+    Given user is at Developer perspective
+    When user opens Monitoring 
+    Then user will see Dashboard tab
+    And user will see Metrics tab
+    And user will see Alerts tab
+    And user will see Events tab
+
+
+@smoke, @regression
+Scenario: Dashboard tab on the Monitoring page for all workloads: MH-01-TC03
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Dashboard tab
+    Then user will see All Workloads
+    And user will see the CPU Usage
+    And user will see the Memory Usage
+    And user will see Receive Bandwidth
+    And user will see Transmit Bandwidth
+    And user will see the Rate of Received Packets
+    And user will see the Rate of Trasmitted Packets
+    And user will see the Rate of Received Packets Dropped
+    And user will see the Rate of Trasmitted Packets Dropped
+
+
+@smoke, @regression
+Scenario: Dashboard tab on the Monitoring page for particular workload: MH-01-TC03
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Dashboard tab
+    And user selects the workload from the dropdown
+    Then user will see the CPU Usage
+    And user will see the Memory Usage
+    And user will see Receive Bandwidth
+    And user will see Transmit Bandwidth
+    And user will see the Rate of Received Packets
+    And user will see the Rate of Trasmitted Packets
+    And user will see the Rate of Received Packets Dropped
+    And user will see the Rate of Trasmitted Packets Dropped
+
+
+@smoke, @regression
+Scenario: Events tab on the Monitoring Page: MH-01-TC05
+    Given user is at Developer perspective
+    When user opens Monitoring page
+    And user clicks on Events tab
+    Then user will events related to all resources and all types
+
+
+@smoke, @regression
+Scenario: Events for Multiple Resources: MH-01-TC05
+    Given user is at Developer perspective
```

Comment:
Same as above and can other given statements provided below this statement can be handled this way

Reply:
Updated.